src/dmd/cparse.d

```diff
@@ -1532,6 +1532,11 @@ final class CParser(AST) : Parser!AST
                     break;
             }
 
+            if (specifier.ealign && dt.isTypeFunction())
+                error("no alignment-specifier for function declaration"); // C11 6.7.5-2
```

Comment:
This error message is not good. You have previously mentioned that you don't want to focus on error messages for `ImportC`, fair enough, however in this case this text doesn't really tell me anything about why the error has been emitted - this wording suggests there isn't an alignment-spec present, rather than that there is when there shouldn't be.

`Error: alignment specified for function 'mercury'` is what GCC does for example.

Reply:
Right now, I'm concentrating on getting the C implementation right. Better error messages come later. Though, of course, anyone can submit PRs to improve messages.